metrics-scraper/monitoring.go

```diff
@@ -69,16 +70,49 @@ func Start(configPath string) {
 	wg.Wait()
 }
 
+// tryConnect, try to connect to the node forever until it works
+// it'll stop when it acquire a connection or it's asked to stop
+func tryConnect(
+	ctx context.Context,
+	wg *sync.WaitGroup,
+	config NodeConfig,
+) (c *rpc.Client, err error) {
+	c, err = rpc.DialContext(ctx, config.RPCURL)
+	if err == nil {
+		// no errors, let's start polling
+		return
+	}
+
+	ticker := time.NewTicker(5 * time.Second) // arbitrarily retry every 5 seconds
```

Comment:
Let's just use the same `pollRate` from the config as for the regular scraping.

Reply:
addressed in: d1755782d3115b584b67e6fb763dc3d62386ef81